packages/contracts-bedrock/contracts/L1/SystemConfig.sol

```diff
@@ -207,4 +217,15 @@ contract SystemConfig is OwnableUpgradeable, Semver {
             sstore(slot, _unsafeBlockSigner)
         }
     }
+
+    /**
+     * @notice Returns the minimum gas limit allowed by the system. If the L2
+     *         gas limit is set to a value smaller than this, then it is
+     *         possible for a block to be produced that uses more gas than what
+     *         is allowed on L2, resulting in a liveliness failure.
+     */
+    function _minimumGasLimit() internal view returns (uint256) {
```

Comment:
Should we expose this as a public method? That would align with the vision of the SystemConfig as the central location for all system info.

Reply:
No super opinionated, can make it into an external function